#!/usr/bin/env python
PACKAGE = "legged_controllers"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("kp_position",    double_t,    0,  "Kp_position",    10,    0,   300)
gen.add("kd_position",    double_t,    1,  "Kd_position",    3,    0,   100)
gen.add("kp_big_stance",    double_t,    2,  "kp_big_stance",    40,    0,   100)
gen.add("kp_big_swing",    double_t,    3,  "kp_big_swing",    30,    0,   100)
gen.add("kd_big",    double_t,    4,  "kd_big",    2,    0,   20)

gen.add("kp_small_stance",    double_t,    5,  "kp_small_stance",    30,    0,   100)
gen.add("kp_small_swing",    double_t,    6,  "kp_small_swing",    20,    0,   100)
gen.add("kd_small",    double_t,    7,  "kd_small",    2,    0,   20)

gen.add("kd_feet",    double_t,    10,  "kd_feet",    0.01,    0,   20)

exit(gen.generate(PACKAGE, "legged_controllers", "Tutorials")) 